kable.go

```diff
@@ -1,24 +1,54 @@
 package kable
 
-import "fmt"
-
-// import (
-// 	"github.com/Kable-io/kable-go/internal/openapi"
-// )
+import (
+	"fmt"
+	"github.com/Kable-io/kable-go/internal/openapi"
+	"github.com/Kable-io/kable-go/internal/version"
+	"net/http"
+	"net/url"
+	"time"
+)
 
 type (
 	KableOptions struct {
+		KableClientId     string
+		KableClientSecret string
+		Debug             bool
+		MaxQueueSize      int
+		//RecordAuthentication bool
+
+		BaseUrl    *url.URL
+		HTTPClient *http.Client
 	}
 
 	Kable struct {
-		Event *Event
+		EventsApi *EventsApi
 	}
 )
 
-func (p Kable) Foo() {
-	fmt.Println("Hello World!")
+var defaultHTTPClient = &http.Client{
+	Timeout: 60 * time.Second,
 }
 
-func New() *Kable {
-	return &Kable{}
+func New(options *KableOptions) *Kable {
+	conf := openapi.NewConfiguration()
+	conf.Scheme = "https"
+	conf.Host = "test.kable.io"
```

Comment:
let's force all customers to define this explicitly (so there's no oversight in production)